storages/merger/bucket_meta_data.go

```diff
@@ -17,17 +22,22 @@ func MergePartially(firstResponse backend.Response, successes []backend.Response
 	if err != nil {
 		log.Printf("Could not read first response body reqID %s, reason %s", firstResponse.ReqID(), err)
 	}
-	firstResponse.DiscardBody()
-	statusCode := http.StatusOK
+	err = firstResponse.DiscardBody()
+	if err != nil {
+		log.Printf("Cannot close first response body reqId %s, reason: %s", firstResponse.ReqID(), err)
```

Comment:
`Cannot close` -> `Cannot discard`
And does `DiscardBody` closes the `body` in case it can't be discarded?

Reply:
Tries to, if ends up with error returns it